lib/widgets/per_account_settings.dart

```diff
@@ -0,0 +1,215 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+import '../basic.dart';
+import '../model/store.dart';
+import 'store.dart';
+
+/// A builder function for [RemoteSettingBuilder.builder]
+/// that creates a toggle, or radio buttons, etc.
+///
+/// [value] is the value, whether from [RemoteSettingBuilder.findValueInStore]
+/// or from local echo.
+///
+/// [handleRequestNewValue] calls [RemoteSettingBuilder.sendValueToServer]
+/// and starts or extends a local-echo period.
+///
+/// If extending a local-echo period, replaces the old local-echo value.
+///
+/// This may be called any time. API requests are not debounced,
```

Comment:
"this" should be [handleRequestNewValue], right? Not the function whose type this is documenting.